go.mod

```diff
@@ -4,6 +4,7 @@ go 1.23.0
 
 require (
 	github.com/cenkalti/backoff/v4 v4.3.0
+	github.com/cenkalti/backoff/v5 v5.0.2
```

Comment:
We have a couple places in the code where we're using backoff/v4. Looking into refactoring those to use v5 instead so we only need a single module that does this instead of pulling in both.